.gitignore

```diff
@@ -1,2 +1,3 @@
+.DS_Store
 .idea
-.csv
+*.csv
```

Comment:
I think adding .ipynb_checkpoints/* (and removing the file from the repo) may be prudent.

Reply:
Oops clearly didn’t `git rm —cached` ha thanks for checking. Will update when home.

Reply:
Oops clearly didn’t `git rm —cached` ha thanks for checking. Will update when home.

Reply:
updated!